mslib/msui/autoplot_dockwidget.py

```diff
@@ -298,7 +298,12 @@ def add_to_treewidget(self, parent, parent2, config_settings, treewidget, flight
             config_settings["automated_plotting_flights"].append([flight, sections, vertical, filename, itime, vtime])
             parent.refresh_signal_emit.emit()
         if treewidget.objectName() == "autoplotSecsTreeWidget":
-            if url is None:
+            if url == "":
+                QMessageBox.information(
+                    self,
+                    "WARNING",
+                    "Please select the URL,layer,styles and level(row information first)"
```

Comment:
the text linter has not seen that, but we should do it :)

Please select the URL, layer, styles and level (row information first)"

Reply:
Done